javascript/implementation/CommandLineInterface.ts

```diff
@@ -12,87 +12,86 @@ import { start, REPLServer } from "node:repl";
 /**
     Intended to manage server side running of Gink.
     Basically it takes some settings in the form of
-    environment variables plus a list of peers to
+    arguments plus a list of peers to
     connect to then starts up the Gink Instance,
     or Gink Server if port listening is specified.
-    TODO(https://github.com/google/gink/issues/43): implement --help
 */
 export class CommandLineInterface {
     targets: string[];
     store?: Store;
     instance?: Database;
     routingServer?: RoutingServer;
     replServer?: REPLServer;
+    authToken?: string;
 
-    constructor(process: NodeJS.Process) {
+    constructor(args: {
+        connect_to?: string[],
+        listen_on?: string,
+        data_root?: string,
+        data_file?: string,
```

Comment:
FYI this library is a direct port from the python one with very few changes. That is why snake_case has made it in here.